frontend/local.go

```diff
@@ -14,8 +14,9 @@ import (
 	"github.com/TBD54566975/ftl/internal/log"
 )
 
-var consoleURL, _ = url.Parse("http://localhost:5173")
-var proxy = httputil.NewSingleHostReverseProxy(consoleURL)
+var proxyURL, _ = url.Parse("http://localhost:5173")
+var publicURL, _ = url.Parse("http://localhost:8892")
```

Comment:
@alecthomas would it be safer to pass in the controller's `Bind` url and adjust the port for the console? Or will these `localhost` definitions work?

Reply:
Yeah that would be preferable, I actually just came back to the PR to suggest exactly that. It won't matter for 5173 obviously.